File: Data Structures & Algorithms/two-integer-sum/submission-11.java
class Solution {
    public int[] twoSum(int[] nums, int target) {

        //   for i in range(len(nums)):
        //     for j in range(i+1,len(nums)):
        //         if nums[i]+ nums[j] ==target and i!=j:
        //             return [i,j]

    
        for(int i=0;i<nums.length;i++){
            for(int j=i+1;j<nums.length;j++){
                if(nums[i]+ nums[j] == target )
                {
                    return new int[]{i,j};
                } 
            }
        }
        return new int[0];
    }
    }   
    

File: Data Structures & Algorithms/single-number/submission-11.py
class Solution:
    def singleNumber(self, nums: List[int]) -> int:
        # x=set()
        # for i in nums:
        #     if i in x:
        #         x.remove(i)
        #     else:
        #         x.add(i)
        
        # return list(x)[0]

        xor=0
        for i in nums:
            xor^=i
        return xor

